documentation/modules/proc-iot-api.adoc

```diff
@@ -13,17 +18,22 @@ endif::[]
 
 = Sending and receiving {api} data
 
-.Procedure
+== Starting the {api} consumer
 
-== Starting {api} consumer
+.Prerequisites
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#installing-services-iot[Installed the IoT services]
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-creating-project-iot[Created an IoT project]
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-creating-device-iot[Created an IoT device]
+
+.Procedure
 
-. In Hono project run:
+. In the Hono project, run:
 +
 [options="nowrap",subs="attributes"]
 ----
 cd cli
 ----
-. At least once run:
+. At least once, run:
```

Comment:
Maybe something like, "Run the following command at least once:"

Still, it's not clear what "at least once" means. How do I know if I need to run it more than once?